protected/components/OEPatientEventDateValidator.php

```diff
@@ -22,6 +22,9 @@ class OEPatientEventDateValidator extends OEDatetimeValidator
 
     public function validateAttribute($object, $attribute)
     {
+        if ($object->$attribute == "0000-00-00"){
```

Comment:
Does this also allow partial dates (eg. only year entered)?

Reply:
Seems no. Need find another way to validate it.